src/solvers/solvers_parabolic.jl

```diff
@@ -0,0 +1,38 @@
+"""
+  ViscousFormulationBassiRebay1()
+
+The classical BR1 flux from
+
+- F. Bassi, S. Rebay (1997)
+  A High-Order Accurate Discontinuous Finite Element Method for
+  the Numerical Solution of the Compressible Navier-Stokes Equations
+  [DOI: 10.1006/jcph.1996.5572](https://doi.org/10.1006/jcph.1996.5572)
+"""
+struct ViscousFormulationBassiRebay1 end
+
+# no penalization for a BR1 parabolic solver
+function calc_viscous_penalty!(scalar_flux_face_values, u_face_values, t, boundary_conditions,
+                               mesh, equations::AbstractEquationsParabolic, dg::DGMulti,
+                               dg_parabolic::ViscousFormulationBassiRebay1, cache, cache_parabolic)
+  return nothing
+end
```

Comment:
Shouldn't this function live in the `dgmulti` subfolder?

Reply:
Yeah, probably - thanks for catching it.